OpenRA.Mods.TS/Traits/Buildings/TiberianSunRefinery.cs

```diff
@@ -11,22 +11,22 @@
 using OpenRA.Activities;
 using OpenRA.Mods.RA.Traits;
 using OpenRA.Mods.TS.Activities;
-using OpenRA.Traits;
 
 namespace OpenRA.Mods.TS.Traits
 {
-	public class TiberianSunRefineryInfo : OreRefineryInfo
+	public class TiberianSunRefineryInfo : RefineryInfo
 	{
 		public override object Create(ActorInitializer init) { return new TiberianSunRefinery(init.Self, this); }
 	}
 
-	public class TiberianSunRefinery : OreRefinery
+	public class TiberianSunRefinery : Refinery
 	{
 		public TiberianSunRefinery(Actor self, TiberianSunRefineryInfo info) : base(self, info) { }
 
 		public override Activity DockSequence(Actor harv, Actor self)
 		{
-			return new VoxelHarvesterDockSequence(harv, self);
+			// TODO: Pass DockAngle from YAML
+			return new VoxelHarvesterDockSequence(harv, self, 160, false);
```

Comment:
Why don't you? You can take the value for the `dragRequired` arg from yaml here, too.